src/app/Pages/ResultsPage/index.jsx

```diff
@@ -73,6 +73,11 @@ export default class ResultsPage extends Component {
     });
   };
 
+  goBack() {
```

Comment:
I know this is copied from somewhere 👀 
can we make this a helper method and import it whenever we need it?